### DISCUSSION TITLE
Why are data not published to mqtt server, when internet is down?

### DISCUSSION THREAD

**Discussion by drlauridsen** [UtilitechAS/amsreader-firmware#533] [Q&A]
My internet has been quite unstable lately, due to ISP problems and router issues.
Every time my internet connection is down, but wifi/LAN still works, data from amsreader on an esp32 is lost. I have a server on the LAN with an mqtt server and Homeassistant, and all energy data are missing from the periods without internet.
Just trying to understand why this is the case - I would assume that  data would still be send within the LAN, regardless of WAN status?
Is it due to missing timestamps from the NTP server?

**Comment by gskjold** [comment]
This is a good question, I will need to set up a test locally to check. But I see no reason why it shouldn't publish to MQTT during internet outage.
